manifests/refresh_pattern.pp

```diff
@@ -1,3 +1,67 @@
+# @summary
+#   Defines refresh_pattern entries for a squid server.
+# @see http://www.squid-cache.org/Doc/config/refresh_pattern/
+# @example
+#   squid::refresh_pattern { '^ftp:':
+#     min     => 1440,
+#     max     => 10080,
+#     percent => 20,
+#     order   => 60,
+#   }
+#
+#   squid::refresh_pattern { '(/cgi-bin/|\?)':
+#     case_sensitive => false,
+#     min            => 0,
+#     max            => 0,
+#     percent        => 0,
+#     order          => 61,
+#   }
+#
+#   would result in the following squid refresh patterns:
+#   # refresh_pattern fragment for ^ftp
+#   refresh_pattern ^ftp: 1440 20% 10080
+#   # refresh_pattern fragment for (/cgi-bin/|\?)
+#   refresh_pattern (/cgi-bin/|\?) -i 0 0% 0
```

Comment:
This doesn't render that well I think since examples are assumes to be puppet code.